src/ESPEasy.ino

```diff
@@ -773,24 +786,26 @@ void setSystemTimer(unsigned long timer, byte plugin, byte Par1, byte Par2, byte
       firstAvailable = x;
     }
   }
-  if (!reUse)
+  if (firstAvailable == SYSTEM_TIMER_MAX )
   {
-    if (firstAvailable == SYSTEM_TIMER_MAX )
-    {
-      addLog(LOG_LEVEL_ERROR, F(NOTAVAILABLE_SYSTEM_TIMER_ERROR));
-    }
-    else
-    {
-      systemTimers[firstAvailable].timer = millis() + timer;
-      systemTimers[firstAvailable].plugin = plugin;
-      systemTimers[firstAvailable].Par1 = Par1;
-      systemTimers[firstAvailable].Par2 = Par2;
-      systemTimers[firstAvailable].Par3 = Par3;
-    }
+    addLog(LOG_LEVEL_ERROR, F(NOTAVAILABLE_SYSTEM_TIMER_ERROR));
   }
+  else
+  {    
+    systemTimers[firstAvailable].plugin = plugin;
+    systemTimers[firstAvailable].TaskIndex = taskIndex;
+    systemTimers[firstAvailable].Par1 = Par1;
+    systemTimers[firstAvailable].Par2 = Par2;
+    systemTimers[firstAvailable].Par3 = Par3;
+    systemTimers[firstAvailable].Par4 = Par4;
+    systemTimers[firstAvailable].Par5 = Par5;
+    systemTimers[firstAvailable].timer = timer > 0
```

Comment:
i made this change, because when i call setSystemTimer with parameter timer set 0, before timer stopping  wait for the first checkSystemTimers call and fire PLUGIN_TIMER_IN event .
This is correct behavior or bug?